test/e2e/validators.go

```diff
@@ -182,6 +182,9 @@ func validateController(
 		return testBuildRun.Status.Succeeded
 	}, 550*time.Second, 5*time.Second).Should(Equal(trueCondition), "BuildRun did not succeed")
 
+	completionTime, err := time.Parse(time.RFC3339Nano, testBuildRun.Status.CompletionTime.String())
+	startTime, err := time.Parse(time.RFC3339Nano, testBuildRun.Status.StartTime.String())
+	Logf("Test build executed '%T' seconds", int(completionTime.Sub(startTime).Seconds()))
```

Comment:
Just a minor wording thing: I personally try to omit the usage of the word `execute` (although I still use it from time to time). The reason is the alternate meaning of it (= to kill somebody). I try to use `run` instead.

Reply:
Hi @SaschaSchwarze0 ,

I refined and verifed my new code:
https://github.com/redhat-developer/build/pull/193/files#diff-49934e1a92fa22365f4fe19df5e529cbR185

If the buildrun is executed correctly, it will return the complate and start time, then we can know the execution time adn report in the end of each buildrun.

You can see the final result below